languages/tgmpa-vi_VN.po

```diff
@@ -1,7 +1,9 @@
+# Translation of TGM Plugin Activation in Vietnamese (VN)
+# This file is distributed under the same license as the TGMPA package.
 msgid ""
 msgstr ""
 "Plural-Forms: nplurals=1; plural=0;\n"
-"Project-Id-Version: TGM Plugin Activation by BBLand\n"
+"Project-Id-Version: TGM Plugin Activation v2.6.1 by BBLand.net\n"
```

Comment:
Please remove the "by BBLand.net" bit. You didn't create TGMPA; your credits for translating are in the other headers.